packages/extension-ui/src/components/Main.tsx

```diff
@@ -21,7 +21,6 @@ export default styled(Main)(
   color: ${theme.textColor};
   font-size: ${theme.fontSize};
   line-height: ${theme.lineHeight};
-  border: 1px solid ${theme.inputBorderColor};
```

Comment:
This border around the entire window was causing the glow by being blured with the filter, but was otherwise practically invisible to the user, so I had no trace of remorse to delete it.